src/pages/Discover.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { downloadMovieAction } from '../store/actions/moviesAction';
+import MoviesList from '../components/movies/MoviesList';
+
+const Discover = () => {    
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        const getDiscover = () => dispatch(downloadMovieAction());
+        getDiscover();
+    }, [dispatch]);    
+
+    const {movies,error} = useSelector(state=>state.movies);        
+
+    return (         
+        <div className="mt-4">
+            <h1>Discover</h1>
+            {
+                !error && movies ? (
+                <MoviesList items={movies.discover} /> 
+                ) : (
+                    <h1>hay error</h1>
+                )
+                
+            }
```

Comment:
This can be improved by extracting a [HOC](https://reactjs.org/docs/higher-order-components.html) to handle conditional logic when there is some content, a possible error and loading state:

```jsx
<LoadingWrapper error={error} loading={loading}>
  <MovieList items={movies.discover} />
</LoadingWrapper>
```

I assume the error message is still WIP but keep the language consistent when possible.

Reply:
Maybe even add the possibility to render the error in different ways:

```jsx
<LoadingWrapper error={error} renderError={(errorMessage) => <span className="text-warning">{errorMessage}</span>}>
```

Reply:
this point remains for the next PR